app/models/status.rb

```diff
@@ -29,8 +29,6 @@
 #
 
 class Status < ApplicationRecord
-  before_destroy :unlink_from_conversations
```

Comment:
@ClearlyClaire 

I moved this because all the other model hooks were at line 307 - 320.  
Only because I had to touch it anyway…

Reply:
I've just discovered that this seemed to have been there for a reason. Specs failed because of that.

I have no Idea why that is, but I guess a non working change will always have to loose against a working status quo. :wink:

So that change is goneagain now.

Reply:
The order in which callbacks are registered matters and I suppose one of the concerns included below somehow conflicts. Haven't looked into it though so I'm not sure exactly what is happening there.